### PR TITLE
fix dd-trace-ot dependency on okio

### PR DESCRIPTION
**Pull request by richardstartin**

### CODE REVIEW

File: dd-trace-ot/dd-trace-ot.gradle [hunk 1]
```diff
@@ -41,6 +41,7 @@ dependencies {
   compile group: 'io.opentracing.contrib', name: 'opentracing-tracerresolver', version: '0.1.0'
 
   compile deps.slf4j
+  compile deps.okio
```
Comment (tylerbenson) [inline dd-trace-ot/dd-trace-ot.gradle:44]: It's not clear why this is needed... why isn't the correct version being picked up from `dd-trace-core`?

Comment (drodriguezhdez) [reply dd-trace-ot/dd-trace-ot.gradle:44]: It seems `dd-trace-core` is not an explicit dependency in `dd-trace-ot`?

image

https://github.com/drodriguezhdez/datadog-example

Comment (tylerbenson) [reply dd-trace-ot/dd-trace-ot.gradle:44]: It's not... it gets shadowed in since dd-trace-core is not published to maven.

**Comment by richardstartin** [comment]
this may not be a solution to the problem

**Comment by richardstartin** [comment]
maybe it does

**Comment by randomanderson** [comment]
#1651

**Pull request merged**